packages/react/menu/src/Menu.stories.tsx

```diff
@@ -235,32 +235,46 @@ export const Typeahead = () => (
 );
 
 export const CheckboxItems = () => {
-  const checkboxItems = [
-    { label: 'Bold', state: React.useState(false) },
-    { label: 'Italic', state: React.useState(true) },
-    { label: 'Underline', state: React.useState(false) },
-    { label: 'Strikethrough', state: React.useState(false), disabled: true },
+  const options = [
+    { label: 'Crows' },
+    { label: 'Ravens' },
+    { label: 'Magpies' },
+    { label: 'Jackdaws' },
```

Comment:
nit: If we're only going to deal with the labels now, let's just make it an array of strings instead.